doc/user/content/manage/user-management/invite-users.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Invite users"
+description: "Invite new organization members or administrators"
+menu:
+  main:
+    parent: user-management
+    weight: 10
+---
+
+This page outlines the steps for inviting a new user. You can invite new users
+to your Materialize organization and assign **Admin** or **Member** privileges.
+
+-> **Note:** You must be an Admin in your organization to invite new users.
+
+## Step 1. Create a new user
+
+1. [Login to the Materialize console](https://console.materialize.com/) and navigate to Account > Account
+Settings > Users.
+
+1. Click **Invite User** and fill in the user information.
+
+![]Image
+
+## Step 2. Determine user Organization privileges
+
+The **Organization Admin** and **Organization Member** roles refer to `SUPERUSER`
```

Comment:
I don't really understand what we're trying to say her. Is `SUPERUSER` an actual thing in Materialize, or are we referencing the concept of `SUPERUSER` in Postgres? https://www.postgresql.org/docs/current/sql-createrole.html.

Also, the overall step sequence here is a bit confusing because you choose a role after entering the user email on the form. 

<img width="666" alt="Screenshot 2023-06-12 at 11 59 52 AM" src="https://github.com/MaterializeInc/materialize/assets/15893490/9d67d949-cae6-4e39-a037-47e404636923">

For this particular how-to, I don't think we need headings. I think it's ok to just use steps like you have above. I'd expand that to something like:

```
1. [Login to the Materialize console](https://console.materialize.com/).

1. Navigate to Account > Account Settings > Users.

1. Click **Invite User**.

1. Fill in the user information.

    Be sure to select the appropriate role for the user:
    
    - **Organization Member**: <What this role means and when to choose it.>
    - **Organization Admin** <What this role means and when to choose it>

1. Click **Invite**.
```

Then say something about what happens. 

Also, it seems that you can choose more one role for a user. Why would you do that? We need to be explicit about this. 

I'm confused about the "Invite with Copy" options and started a Slack thread in #feedback.